python/lsst/sconsUtils/scripts.py

```diff
@@ -346,9 +346,13 @@ def python(swigNameList=None, libs="main python", swigSrc=None):
     #  @param extraSrc    A dictionary of additional source files that go into the modules.  Each
     #                     key should be an entry in nameList, and each value should be a list
     #                     of additional C++ source files.
+    #  @param addUnderscore  Add an underscore to each library name (if the source file name
+    #                     does not already start with underscore)? If false the library name
+    #                     is always the same as the source file name
+    #                     DEPRECATED: always use False for new code.
```

Comment:
Hmm, not sure I like having to always specify `False` just to bridge a small gap before the merge.